pom.xml

```diff
@@ -118,6 +118,11 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-configuration-processor</artifactId>
```

Comment:
Yes, but if all it does is to generate auto-completion in the properties in an specific IDE, I'd say that the decision about whether this is required or not belongs to the end user of this library.